examples/c/chain_model/chain_model.h

```diff
@@ -7,23 +7,23 @@
 extern "C" {
 #endif
 
-#define X0_NM2_FILE "/home/bnovoselnik/Software/acados/examples/c/chain_model/x0_nm2.txt"
-#define X0_NM3_FILE "/home/bnovoselnik/Software/acados/examples/c/chain_model/x0_nm3.txt"
-#define X0_NM4_FILE "/home/bnovoselnik/Software/acados/examples/c/chain_model/x0_nm4.txt"
-#define X0_NM5_FILE "/home/bnovoselnik/Software/acados/examples/c/chain_model/x0_nm5.txt"
-#define X0_NM6_FILE "/home/bnovoselnik/Software/acados/examples/c/chain_model/x0_nm6.txt"
-#define X0_NM7_FILE "/home/bnovoselnik/Software/acados/examples/c/chain_model/x0_nm7.txt"
-#define X0_NM8_FILE "/home/bnovoselnik/Software/acados/examples/c/chain_model/x0_nm8.txt"
-#define X0_NM9_FILE "/home/bnovoselnik/Software/acados/examples/c/chain_model/x0_nm9.txt"
-
-#define XN_NM2_FILE "/home/bnovoselnik/Software/acados/examples/c/chain_model/xN_nm2.txt"
-#define XN_NM3_FILE "/home/bnovoselnik/Software/acados/examples/c/chain_model/xN_nm3.txt"
-#define XN_NM4_FILE "/home/bnovoselnik/Software/acados/examples/c/chain_model/xN_nm4.txt"
-#define XN_NM5_FILE "/home/bnovoselnik/Software/acados/examples/c/chain_model/xN_nm5.txt"
-#define XN_NM6_FILE "/home/bnovoselnik/Software/acados/examples/c/chain_model/xN_nm6.txt"
-#define XN_NM7_FILE "/home/bnovoselnik/Software/acados/examples/c/chain_model/xN_nm7.txt"
-#define XN_NM8_FILE "/home/bnovoselnik/Software/acados/examples/c/chain_model/xN_nm8.txt"
-#define XN_NM9_FILE "/home/bnovoselnik/Software/acados/examples/c/chain_model/xN_nm9.txt"
+#define X0_NM2_FILE "/Users/robin/acados/examples/c/chain_model/x0_nm2.txt"
```

Comment:
hahahah. We need to find a good solution with this stupid file

Reply:
the good solution it to remove it... at least in make, it is not used any longer (I don't know about cmake)

Reply:
really? but how do you do it then? Cmake also does not need it as it code generates it from the .h.in file so we could remove it from the repo and put it in the ignore list

Reply:
I hand-coded the .txt files
https://github.com/acados/acados/blob/master/examples/c/chain_model/xN_nm2.txt
into .c files
https://github.com/acados/acados/blob/master/examples/c/chain_model/xN_nm2.c
which are included in the examples
https://github.com/acados/acados/blob/master/examples/c/nonlinear_chain_ocp_nlp.c#L51

But this is not automated in the casadi script to generate data, which would change only the .txt files.